src/tf/tf_parser.cpp

```diff
@@ -0,0 +1,617 @@
+#include <google/protobuf/text_format.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <graph.pb.h>
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <unordered_set>
+#include <functional>
+#include <array>
+#include <utility>
+#include <vector>
+
+#include <migraphx/fallthrough.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/op/argmax.hpp>
+#include <migraphx/op/argmin.hpp>
+#include <migraphx/op/batch_norm_inference.hpp>
+#include <migraphx/op/common.hpp>
+#include <migraphx/op/concat.hpp>
+#include <migraphx/op/convolution.hpp>
+#include <migraphx/op/gather.hpp>
+#include <migraphx/op/pad.hpp>
+#include <migraphx/op/pooling.hpp>
+#include <migraphx/op/reshape.hpp>
+#include <migraphx/op/slice.hpp>
+#include <migraphx/op/softmax.hpp>
+#include <migraphx/op/squeeze.hpp>
+#include <migraphx/op/transpose.hpp>
+#include <migraphx/op/unknown.hpp>
```

Comment:
I dont think all these op headers are needed to be included.

Reply:
oh yeah I forgot to remove those when refactoring.